CONTRIBUTING.md

```diff
@@ -277,4 +278,120 @@ Your name will likely be included in the Git commit log. If your change affects
 end users, a brief description and your name might be added to the [NEWS](/NEWS)
 file.
 
+## Git commit rules
+
+This is the first file you should be reading when contributing code via Git.
```

Comment:
This sentence is somewhat odd when it's in the middle of the file...

Generally I think it should be made more obvious that this section refers to contributors that already have commit access and commit changes themselves. The title should probably be changed to make that obvious.

Reply:
Indeed. Hopefully the next commit makes this a bit more clear...